Assets/Scripts/BlockScripts/nextLetter.cs

```diff
@@ -0,0 +1,39 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using Letter;
+
+public class nextLetter : MonoBehaviour
+{
+    private static Random _rnd = new Random();
+
+    public static Letter alphabet(List<string> alphabet, int totalWeight);
+
+    // Start is called before the first frame update
+    void Start()
+    {
+        alphabet = createAlphabet(language);
+    }
+
+    // Update is called once per frame
+    void Update()
```

Comment:
I'd move the logic out of the update method, Update has no return type, and it will attempt to return a letter once every frame, could be upto 200 times per second

Reply:
Moved out to a new function